evap/evaluation/templates/startpage_button.html

```diff
@@ -0,0 +1,25 @@
+{% if user.startpage == page %}
+    <div data-bs-toggle="tooltip" data-bs-placement="left" title="{% trans 'This is your startpage' %}">
+        <button type="button" class="btn btn-sm btn-light" disabled><span class="fas fa-house"></span></button>
+    </div>
+{% else %}
+    <button id="makeStartPageButton" type="button" class="btn btn-sm btn-light" onclick="makeStartPage();" data-bs-toggle="tooltip" data-bs-placement="left" title="{% trans 'Make this my startpage' %}"><span id="span-set-startpage" class="fas fa-house"></span></button>
+{% endif %}
+
+
+{% block additional_javascript %}
+    <script type="text/javascript">
+        function makeStartPage() {
+            var makeStartPageButton = $('#makeStartPageButton');
```

Comment:
In spirit of our current motto of preferring browser behavior over javascript and trimming down on jQuery in particular, I would suggest to replace the current logic with a form. The button becomes a submit with `name="page" value="{{ page }}"` and the view redirects to the users start page - which, if updating the startpage worked correctly, should be equivalent to reloading the current site